### ISSUE TITLE
Make the App Responsive

### ISSUE THREAD

**Issue by niloysikdar** [niloysikdar/React-Meetups#6]
Currently, it is only made for Desktop view. Need to add responsiveness for small screen devices using proper CSS.

**Comment by wise-introvert** [comment]
@niloysikdar Have you considered TailwindCSS? It'll make it a whole lot easier to implement responsiveness.

( Sorry if this isn't helpful :sweat_smile: )

**Comment by niloysikdar** [comment]
> @niloysikdar Have you considered TailwindCSS? It'll make it a whole lot easier to implement responsiveness.
> 
> ( Sorry if this isn't helpful 😅 )

Yes, @wise-introvert , I have used TailwindCSS and bootstrap in my projects, will definitely use them if I don't get enough time to write own CSS.

**Comment by misztijanos** [comment]
Hi, are you looking for contributors on this issue? If so, I'd be happy to work on it.

**Comment by niloysikdar** [comment]
Hey @misztijanos , contributions are always welcomed. I will be more than happy if someone makes a valid contribution to open source. Cheers

**Comment by misztijanos** [comment]
Good to hear. Any constraints, guidelines, you wish to add? Or should I just use my judgement?

**Comment by niloysikdar** [comment]
Anything you want @misztijanos , you can either use your own CSS code or else you can also use any other CSS framework.
